net/atlas-sw-probe/files/atlas.readme

```diff
@@ -0,0 +1,12 @@
+# Atlas probe setup instruction

```

Comment:
I am wondering if we want to ship this file together with the file as it seems we are supplying documentation. Well, there should be, I know, but in my view, I can copy&paste [the documentation](https://wiki.turris.cz/doc/en/howto/atlas-probe) which I wrote together with @ja-pa and move it to OpenWrt's documentation.

Reply:
However, we can solve it later. Let's dont delay merging this. :-)